src/engine/DebugOverlays.EntityLabel.cs

```diff
@@ -32,11 +79,14 @@ private void UpdateEntityLabels()
 
         foreach (var pair in entityLabels)
         {
-            var entity = pair.Key.EntityNode;
+            var iEntity = pair.Key;
+            var entity = iEntity.EntityNode;
```

Comment:
No longer needed.

Reply:
What? Why?

Reply:
Ah wait, I see that this is still used in line 86. Nevermind this.